mountainduck/issues/index.md

```diff
@@ -253,7 +253,7 @@ It may be that Finder closes the connection because Mountain Duck hasn't answere
 
 This is an issue within the operating system that can occur to any network drive.
 
-<del> **Additional `._*` Files Saved on Remote Volumes** </del> <br/>
```

Comment:
Wasn't this formatted with strikethrough because no longer applicable?

Reply:
Strikethrough the header is not very user friendly as it might be difficult to find

Reply:
Then we should rather remove the section as it no longer applies.